### ISSUE TITLE
Problem when importing NodeGraphQt (NodeGraphQt.base)

### ISSUE THREAD

**Issue by LucasCTN** [jchanvfx/NodeGraphQt#108]
Since NodeGraphQt is not on https://pypi.org, i tried to install it on my project by using `pip install https://github.com/jchanvfx/NodeGraphQt/archive/v0.0.3.zip`.

When i try to import NodeGraphQt, I receive this error:

```
File \env\lib\site-packages\NodeGraphQt\__init__.py", line 57, in <module>
    from .base.graph import NodeGraph
ModuleNotFoundError: No module named 'NodeGraphQt.base'
```

This doesn't happen if I clone the project I run from inside it. Maybe it could be a misconfiguration on `setup.py`?

**Comment by LucasCTN** [comment]
I just tried, and it seems like the PR #107 will resolve this issue. It was indeed a misconfiguration on setup.py on installing dependencies.

**Comment by LucasCTN** [comment]
Fixed by the merge of #107. Thanks!
